app/controllers/declaration/RepresentativeDetailsController.scala

```diff
@@ -64,7 +64,7 @@ class RepresentativeDetailsController @Inject()(
     declarationType match {
       case DeclarationType.SUPPLEMENTARY =>
         controllers.declaration.routes.DeclarationAdditionalActorsController.displayPage
-      case DeclarationType.STANDARD | DeclarationType.SIMPLIFIED | DeclarationType.OCCASIONAL =>
+      case DeclarationType.STANDARD | DeclarationType.SIMPLIFIED | DeclarationType.OCCASIONAL | DeclarationType.CLEARANCE =>
         controllers.declaration.routes.CarrierDetailsController.displayPage
       case DeclarationType.CLEARANCE =>
```

Comment:
Plese remove redundant case - it is handled by previouse

Reply:
Fixed this